troposphere/static/js/components/common/InstanceHistoryDetail.react.js

```diff
@@ -0,0 +1,121 @@
+define(function (require) {
+
+  var React = require('react/addons'),
+    Router = require('react-router'),
+    Backbone = require('backbone'),
+    BreadcrumbBar = require('components/projects/common/BreadcrumbBar.react'),
+    InstanceInfoSection = require('components/projects/resources/instance/details/sections/InstanceInfoSection.react'),
+    InstanceDetailsSection = require('components/projects/resources/instance/details/sections/InstanceDetailsSection.react'),
+    InstanceMetricsSection = require('components/projects/resources/instance/details/sections/InstanceMetricsSection.react'),
+    InstanceActionsAndLinks = require('components/projects/resources/instance/details/actions/InstanceActionsAndLinks.react'),
+    stores = require('stores'),
+    Time = require('components/common/Time.react'),
+    ResourceTags = require('components/projects/resources/instance/details/sections/ResourceTags.react'),
+    actions = require('actions'),
+    moment = require('moment'),
+    CryptoJS = require('crypto-js'),
+    Gravatar = require('components/common/Gravatar.react');
+
```

Comment:
We'll get this move to ES6 Module definitions. But, in the future - any *new* components should be defined using the ES6 module import / export syntax: [ES6 Modules](https://gist.github.com/lenards/711efb16001d02152598)